project/Dependencies.scala

```diff
@@ -107,7 +107,7 @@ object Dependencies {
 
   val googleStorageLocal: ModuleID = "com.google.cloud" % "google-cloud-nio" % "0.126.13" % "test"
 
-  val workbenchUtil: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-util" % s"0.6-${workbenchLibsHash}"
+  val workbenchUtil: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-util" % s"0.8-${workbenchLibsHash}"
```

Comment:
I found that no 0.7-d764a9b version exists, despite the fact that the changelog is showing that as a version. I looked at the commits in workbench-libs and don't understand how/why the version bumped up to .8 (FYI @jdcanas).

https://github.com/broadinstitute/workbench-libs/pull/1352/files#diff-4d600ef3b044c2de9f003897736b6540fb361bd2b3b683e6785e37d0137b988eR7